docs/index.md

```diff
@@ -0,0 +1,38 @@
+
+## Membros
+Nome	|GitHub	|E-mail | Disciplina
+--|--|--|--
+Eduardo Nunes Pícolo	|EduardoPicolo	|eduardonpicolo@gmail.com| MDS
+Estevão de Jesus Reis 	|estevaoreis25	|estevaoreis1587@gmail.com| MDS
+Gabriela Medeiros da Silva	|gabiMSilva	|gabriela.medeiros2010@gmail.com| EPS
+Geovana Ramos Sousa Silva	|GeovanaRamos	|geovannna.1998@gmail.com| EPS
+Jaime Juan de Castro Feliciano Damasceno	|JaimeJuan11	|jaime_ruan@hotmail.com| MDS
+Lucas Melo dos Santos	|luucas-melo	|lucasmelodos322@gmail.com| MDS
+Pedro de Miranda Haick	|peHaick	|pedrohaickzz@gmail.com| MDS
+Rodrigo Carvalho dos Santos	|Rocsantos	|ro12062000@gmail.com| MDS
+
+## Tema
+
+Plataforma para suporte aos workshops do Django Girls. O objetivo geral é permitir que as ações realizadas nos workshops possam acontecer remotamente e sem a necessidade de um tempo pré definido de realização, ou seja, as aprendizes podem realizar o tutorial no seu tempo e ter acesso ao suporte de seus mentores por meio da plataforma. Veja mais detalhes em nossos [Documentos](documentos/doc_visao.md).
+
+## Organização do Repositório
+
+**Documentos**<br>
+Reúne as documentações que definem o escopo do projeto, sejam questões de negócio ou definições técnicas.
+
+**Produto**<br>
+Reúne as documentações que detalham o produto e o projeto de forma detalhada.
+
+**Gestão**<br>
+Reúne as documentações referentes às práticas adotadas para o bom funcionamento do projeto.
+
+**Sprints**<br>
+Reúne os relatórios de cada sprint.
+
+## Extras
+
+Alguns detalhes importantes em relação a este projeto:
+
+- Devido à pandemia do COVID-19, as atividades que tradicionalmente seriam feitas presencialmente, como as reuniões de sprint, aconteceram inteiramente no modo remoto. 
+
+- Por ter apenas dois membros de EPS, a divisão de papéis foi feita de uma maneira diferente, considerando a existência de quatro papéis. Mais detalhes em [Roadmaps](produto/roadmaps.md).
```

Comment:
@gabiMSilva Eu coloquei esse link mas na página ainda não existe essa explicação da divisão do papéis. Removo? Adicionamos o texto nessa branch? Em outra?

Reply:
Sim, no README acho que não precisamos entrar em detalhes sobre papeis do time

Reply:
Esse não é o readme. Está na wiki.

Reply:
Pode deixar o link que eu adiciono essa etapa na página de roadmaps